blazor-features/OnChangeEventWithSelectDropdown/OnChangeEventWithSelectDropdown/Pages/Index.razor

```diff
@@ -50,25 +60,25 @@
     public void HandleChange(ChangeEventArgs args)
     {
         @foreach (var topping in toppings)
-        {
-            if (topping.Id == int.Parse(args.Value.ToString()))
             {
-                selectedTopping = topping.Name;
-                totalCost = Math.Round(baseBurgerCost + topping.Price, 2);
+                if (topping.Id == int.Parse(args.Value.ToString()))
```

Comment:
@semirteskeredzic 
Another formatting change. The brackets are not properly aligned.